includes/class-sensei-list-table.php

```diff
@@ -174,6 +174,14 @@ function get_column_info() {
 		$columns = $this->get_columns();
 		$hidden  = get_hidden_columns( $this->screen );
 
+		$sortable_columns = $this->get_sortable_columns();
+
+		$legacy_screen_id = preg_replace( '/^sensei\-lms\_/', 'sensei_', $this->screen->id );
+		if ( has_filter( "manage_{$legacy_screen_id}_sortable_columns" ) ) {
+			_deprecated_hook( esc_html( "manage_{$legacy_screen_id}_sortable_columns" ), '2.0.1', esc_html( "manage_{$this->screen->id}_sortable_columns" ) );
+			$sortable_columns = apply_filters( "manage_{$legacy_screen_id}_sortable_columns", $sortable_columns );
```

Comment:
This triggered a linter issue in the build, because the hook isn't prefixed with the plugin slug. This is one small instance of a much bigger problem since we changed the slug to `sensei-lms`, but for the sake of having a green build before merging, maybe we could add an ignore for this one?

Reply:
@alexsanford I'll add the ignore, but also I intentionally didn't change the `prefixes` property in `phpcs.xml`. I think code-wise we'll always have `sensei` as the de facto prefix.

Reply:
Oooh perfect! I thought it would automatically use the plugin slug. That makes me feel better 😄